packages/manager/apps/catalog-revamp/src/utils/mocks/test-products.mock.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
don't put the mock folder in the src folder

Reply:
Done

Reply:
Unable to move because was trying to put the path @mock in jest.config.js but the CI don't like it, better not touch it to not break the test CI

Reply:
We don't need to have an @mock path actually. That is not important.
Just use a relative path.